core/src/main/java/hudson/Functions.java

```diff
@@ -665,7 +667,25 @@ public static boolean isAutoRefresh(HttpServletRequest request) {
     public static boolean isCollapsed(String paneId) {
     	return PaneStatusProperties.forCurrentUser().isCollapsed(paneId);
     }
-    
+
+    public static boolean useUserTimeZone() {
```

Comment:
Additions to the API either need Javadoc and `@since TODO` (to be filled in later), or `@Restricted(NoExternalUse.class)`